.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
```

Comment:
wondering if we actually need dependabot (there's only a single module we depend on, and pushing it to a newer version also forces consumers of this module to update)

Reply:
Its fine to have... except its already merged so looks like a bad rebase here.

Reply:
Not sure how it ended up here. Rebased, seems fixed now.